Code/Editor/Util/FileUtil.cpp

```diff
@@ -305,6 +305,12 @@ void CFileUtil::EditTextFile(const char* txtFile, int line, IFileUtil::ETextFile
     ExtractFile(fullPathName);
     QString cmd = Platform::MakePlatformFileEditString(fullPathName, line);
 
+    if (fileType == IFileUtil::FILE_TYPE_SYSTEM)
+    {
+        QDesktopServices::openUrl(QUrl::fromLocalFile(fullPathName));
+        return;
+    }
+
     Common::EditFileType editFileType = Common::EditFileType::FILE_TYPE_SCRIPT;
 
     switch (fileType)
```

Comment:
nit -this if could be merged with the switch statement below but it is doing slightly different behavior so I see why it's separated

Reply:
Got a simple way as @cgalvan said.